discovery-infra/test_infra/helper_classes/config/cluster_config.py

```diff
@@ -21,5 +21,8 @@ class BaseClusterConfig(BaseEntityConfig, ABC):
     vip_dhcp_allocation: bool = None
     cluster_networks: List[models.ClusterNetwork] = None
     service_networks: List[models.ServiceNetwork] = None
+    machine_networks: List[models.MachineNetwork] = None
     kubeconfig_path: str = None
     network_type: str = None
+    api_vip: str = None
+    ingress_vip: str = None
```

Comment:
Not sure those 2 need to be in `ClusterConfig` object

Reply:
I don't have a lot of choice here
Once it's in advanced networking configuration we try to save it to this config object